strings/base_collections_vector.h

```diff
@@ -191,6 +198,18 @@ namespace winrt::impl
 
     private:
 
+        static auto extract_value(Windows::Foundation::IInspectable const& value)
+        {
+            if constexpr (!std::is_base_of_v<Windows::Foundation::IInspectable, T>)
+            {
+                if (!value)
```

Comment:
Upon further consideration, I think this should go into unbox_value. New iteration coming.